tests/js/spec/views/settings/accountSettingsLayout.spec.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import {mount} from 'enzyme';
+
+import * as OrgActions from 'app/actionCreators/organizations';
+import AccountSettingsLayout from 'app/views/settings/account/accountSettingsLayout';
+
+describe('AccountSettingsLayout', function() {
+  let wrapper;
+  let spy;
+  let api;
+
+  let organization = {
```

Comment:
Are we defining this org to better reflect the "summary" org (vs org details)?

Reply:
Ya that's what I was going for.